geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/CreateIndexCommand.java

```diff
@@ -96,7 +96,7 @@ public ResultModel createIndex(@CliOption(key = CliStrings.CREATE_INDEX__NAME, m
             .createError("Region " + regionName + " does not exist in some of the groups.");
       }
       if (groups == null) {
```

Comment:
I would recommend putting the determination of member for the partition region here.